src/sdk/error.ts

```diff
@@ -27,5 +28,6 @@ export class IntegrationError extends Error {
     this.name = 'IntegrationError';
     this.meta = meta;
     this.cause = meta.cause;
+    this.alert = true;
```

Comment:
why is this true?

Reply:
By default we alert on all integration error. In the useErrorMapping, I'm going to use this property to determine whether to alert or not

Reply:
I can also just do the check in the useErrorMapping as well